sendtweet

```diff
@@ -20,13 +22,8 @@
 # You can also define these values in another file named .twitterkeys
 # by placing it in the same directory.
 
-CONSUMER_KEY='SPg4RX8Zdn5nxzP01IgCQGGQB'
-CONSUMER_SECRET='TmjRpOV6s2ASI5Dh8mBjM3f0s91zh4hemc82S4cbQxHQgRXlng'
-ACCESS_TOKEN='83097255-kg1p0lozz3SVJcP3h8A9IH88AQVx2I5YZBMZn77aC'
-ACCESS_TOKEN_SECRET='Y5VgH8ewwYNjB2GvNgTkEGROnnmne76cwUBYjAGhE4lIU'
-
-if [ -f .twitterkeys ]; then
-    source .twitterkeys
+if [ -f .secret ]; then
```

Comment:
it should be `secret.sh`, that is the Bash file which is processed inline. See https://linuxize.com/post/bash-source-command/